modules/monitoring-configuring-external-alertmanagers.adoc

```diff
@@ -3,144 +3,116 @@
-ifdef::openshift-dedicated,openshift-rosa[]
+
+// tag::CPM[]
+You can add external Alertmanager instances to route alerts for core {product-title} projects.
+// end::CPM[]
+// tag::UWM[]
 You can add external Alertmanager instances to route alerts for user-defined projects.
-endif::openshift-dedicated,openshift-rosa[]
+// end::UWM[]
 
 If you add the same external Alertmanager configuration for multiple clusters and disable the local instance for each cluster, you can then manage alert routing for multiple clusters by using a single external Alertmanager instance.
 
 .Prerequisites
 
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+// end::CPM[]
+// tag::UWM[]
 ifndef::openshift-dedicated,openshift-rosa[]
-* *If you are configuring core {product-title} monitoring components in the `openshift-monitoring` project*:
-** You have access to the cluster as a user with the `cluster-admin` cluster role.
-** You have created the `cluster-monitoring-config` config map.
-* *If you are configuring components that monitor user-defined projects*:
-** You have access to the cluster as a user with the `cluster-admin` cluster role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
-** A cluster administrator has enabled monitoring for user-defined projects.
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+* A cluster administrator has enabled monitoring for user-defined projects.
 endif::openshift-dedicated,openshift-rosa[]
 ifdef::openshift-dedicated,openshift-rosa[]
 * You have access to the cluster as a user with the `dedicated-admin` role.
 * The `user-workload-monitoring-config` `ConfigMap` object exists. This object is created by default when the cluster is created.
 endif::openshift-dedicated,openshift-rosa[]
+// end::UWM[]
 * You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Edit the `ConfigMap` object.
-ifndef::openshift-dedicated,openshift-rosa[]
-** *To configure additional Alertmanagers for routing alerts from core {product-title} projects*:
-.. Edit the `cluster-monitoring-config` config map in the `openshift-monitoring` project:
+. Edit the `{configmap-name}` config map in the `{namespace-name}` project:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+$ oc -n {namespace-name} edit configmap {configmap-name}
 ----
 
-.. Add an `additionalAlertmanagerConfigs:` section under `data/config.yaml/prometheusK8s`.
-
-.. Add the configuration details for additional Alertmanagers in this section:
+. Add an `additionalAlertmanagerConfigs:` section with configuration details under 
```

Comment:
We do not include the ":" when referring to YAML components.

Reply:
Yes, this definitely should not be there and is a typo. Thank you for noticing!